### DISCUSSION TITLE
Credentials_disclosure template

### DISCUSSION THREAD

**Discussion by ratpunk2** [projectdiscovery/nuclei-templates#712] [Q&A] [answered]
First I would like to thank you for this awesome tool.

I am new in BB and today when I was  scanning a website for vulnerability, I got one medium (Credentials_disclosure=www.site.com [sentry_key=, sentry_key;]. I tried to search for the template used to find the possible vuln but couldn't . Please help to recognize the template.

**Comment by ehsandeep** [comment, marked as answer]
Hi @ratpunk2 

You can simply search by template ID to get the template, in this case, it is `credentials-disclosure`

![image](https://user-images.githubusercontent.com/8293321/103224890-ae91ce80-494e-11eb-9583-9dac5b903652.png)

Template link:- https://github.com/projectdiscovery/nuclei-templates/blob/b4b712f4e85cf254a76bfa356fc9d653e176e73a/tokens/credentials-disclosure.yaml

Results from this template always require manual verification to confirm the key validity and its privilege.

**Reply by ratpunk2** [reply]
Thanks a lot sir.
